subgraphs/_reference_/src/prices/index.ts

```diff
@@ -69,58 +69,36 @@ export function getUsdPricePerToken(tokenAddr: Address): CustomPriceType {
     return calculationsSushiSwapPrice;
   }
 
-  // 5. One Inch Oracle
-  let oneInchOraclePrice = getTokenPriceFromOneInch(tokenAddr, network);
-  if (!oneInchOraclePrice.reverted) {
-    log.warning("[OneInchOracle] tokenAddress: {}, Price: {}", [
-      tokenAddr.toHexString(),
-      oneInchOraclePrice.usdPrice
-        .div(oneInchOraclePrice.decimalsBaseTen)
-        .toString(),
-    ]);
-    return oneInchOraclePrice;
-  }
-
-  // 6. Yearn Lens Oracle
-  let aaveOraclePrice = getTokenPriceFromAaveOracle(tokenAddr, network);
+  // 6. Aave Oracle
+  const aaveOraclePrice = AaveOracle.getTokenPriceUSDC(tokenAddr);
   if (!aaveOraclePrice.reverted) {
-    log.warning("[AaveOracle] tokenAddress: {}, Price: {}", [
+    log.info("[AaveOracle] tokenAddress: {}, Price: {}", [
       tokenAddr.toHexString(),
       aaveOraclePrice.usdPrice.div(aaveOraclePrice.decimalsBaseTen).toString(),
     ]);
     return aaveOraclePrice;
   }
 
   // 7. Curve Router
-  let curvePrice = getCurvePriceUsdc(tokenAddr, network);
+  const curvePrice = CurveRouter.getCurvePriceUsdc(tokenAddr);
   if (!curvePrice.reverted) {
-    log.warning("[CurveRouter] tokenAddress: {}, Price: {}", [
+    log.info("[CurveRouter] tokenAddress: {}, Price: {}", [
       tokenAddr.toHexString(),
       curvePrice.usdPrice.div(curvePrice.decimalsBaseTen).toString(),
     ]);
     return curvePrice;
   }
 
   // 8. Uniswap Router
-  let uniswapPrice = getPriceUsdcUniswap(tokenAddr, network);
+  const uniswapPrice = UniswapForksRouter.getTokenPriceUSDC(tokenAddr);
   if (!uniswapPrice.reverted) {
-    log.warning("[UniswapRouter] tokenAddress: {}, Price: {}", [
+    log.info("[UniswapRouter] tokenAddress: {}, Price: {}", [
       tokenAddr.toHexString(),
       uniswapPrice.usdPrice.div(uniswapPrice.decimalsBaseTen).toString(),
     ]);
     return uniswapPrice;
   }
 
-  // 9. SushiSwap Router
-  let sushiswapPrice = getPriceUsdcSushiswap(tokenAddr, network);
-  if (!sushiswapPrice.reverted) {
-    log.warning("[SushiSwapRouter] tokenAddress: {}, Price: {}", [
-      tokenAddr.toHexString(),
-      sushiswapPrice.usdPrice.div(sushiswapPrice.decimalsBaseTen).toString(),
-    ]);
-    return sushiswapPrice;
-  }
-
   log.warning("[Oracle] Failed to Fetch Price, tokenAddr: {}", [

```

Comment:
you should add the token name in there too, it makes it easier to quickly see what token failed

Reply:
Done